tests/io/test_metadata.py

```diff
@@ -111,11 +111,22 @@ def test_save_transformations_incremental(element_name, full_sdata, caplog):
 
 
 # test io for channel names
-@pytest.mark.skip(reason="Not implemented yet")
 def test_save_channel_names_incremental(images: SpatialData) -> None:
-    # note: the non-incremental IO for channel names is already covered in TestReadWrite.test_images(), so here we
-    # only test the incremental IO
-    pass
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        f0 = os.path.join(tmp_dir, "sdata.zarr")
+        images.write(f0)
+
+        new_channels = ["first", "second", "third"]
```

Comment:
after removing the method `set_sdata_image_channel_names()`, this test needs to use `set_channel_names()` followed by `sdata.write_channel_names()`.